txcache/accountBreadcrumb.go

```diff
@@ -66,26 +66,24 @@ func (breadcrumb *accountBreadcrumb) isContinuous(
 	}
 
 	_, ok := sendersInContinuityWithSessionNonce[address]
-	if !ok && !breadcrumb.verifyContinuityWithSessionNonce(accountNonce) {
-		log.Debug("accountBreadcrumb.isContinuous",
+	continuousWithSessionNonce := breadcrumb.verifyContinuityWithSessionNonce(accountNonce)
+	if !ok && !continuousWithSessionNonce {
+		log.Debug("accountBreadcrumb.isContinuous breadcrumb not continuous with session nonce",
 			"address", address,
 			"accountNonce", accountNonce,
-			"breadcrumb nonce", breadcrumb.initialNonce,
-			"err", "breadcrumb not continuous with session nonce")
+			"breadcrumb nonce", breadcrumb.initialNonce)
 		return false
-	} else if !ok {
-		sendersInContinuityWithSessionNonce[address] = struct{}{}
 	}
+	sendersInContinuityWithSessionNonce[address] = struct{}{}
```

Comment:
my suggestion was to only remove the else:
```suggestion
	if !ok{
		sendersInContinuityWithSessionNonce[address] = struct{}{}
	}
```

Reply:
got it, fixed